code/mob/living/critter/small_animal.dm

```diff
@@ -2877,6 +2878,24 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 			src.make_critter(/mob/living/critter/small_animal/mouse/weak)
 			return
 
+/datum/targetable/critter/mentordisappear
+	name = "Vanish"
+	desc = "Leave your body and return to ghost form"
+	icon_state = "mentordisappear"
+
+
+	cast(mob/target)
+
+		var/mob/living/M = holder.owner
+		if (!holder)
+			return 1
+		M.visible_message("<span class='alert'><B>[M] does a funny little jiggle with their body and then vanishes into thin air!</B></span>") // MY ASCENCION BEGINS
```

Comment:
```suggestion
		M.visible_message("<span class='alert'><B>[M] does a funny little jiggle with their body and then vanishes into thin air!</B></span>") // MY ASCENSION BEGINS
```
important

also, don't think this will get me to stop leaving my corpse in funny places when I am done mentoring someone^^